.gitattributes

```diff
@@ -1,6 +1,5 @@
 # Binary data types
 *.aif binary
-*.aifc binary
```

Comment:
I don't think that this change is correct. The ``Lib/test/test_email/data/sndhdr.aifc`` file should still be treated as binary by Git.